process/transaction/shardProcess.go

```diff
@@ -832,20 +935,14 @@ func (txProc *txProcessor) processUserTx(
 	userTx *transaction.Transaction,
 	relayedTxValue *big.Int,
 	relayedNonce uint64,
-	txHash []byte,
+	originalTxHash []byte,
 ) (vmcommon.ReturnCode, error) {
 
 	acntSnd, acntDst, err := txProc.getAccounts(userTx.SndAddr, userTx.RcvAddr)
```

Comment:
is it possible that during the execution of relayed tx we reach this point and we can get an error here?
if so we need extra cleanup.

Reply:
handled the situation. should not happen though